timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -60,6 +60,7 @@ class AbstractTimeSeriesModel(AbstractModel):
     """
 
     _oof_filename = "oof.pkl"
+    _regressor_fit_time_fraction: float = 0.5
```

Comment:
`_covariate_regressor_fit_time_fraction`? Also, if we have this as a class variable, shouldn't we already yhink about some models where it can be higher and override?

Reply:
I don't have an idea for what a reasonable override will be since this significantly depends on the hardware, model type, dataset size, absolute time_limit. 

I will add a TODO so that we think about it at some point in the future.